docker/build.Dockerfile

```diff
@@ -0,0 +1,76 @@
+# backward compatibility:
+FROM ubuntu:12.04.5
```

Comment:
The older the better, this way it also runs on older versions of e.g., debian and CentOS and RHEL and whatnot. But it is up to you to decide which target systems you still want to support. Some application projects build on CentOS 6.x for this reason.